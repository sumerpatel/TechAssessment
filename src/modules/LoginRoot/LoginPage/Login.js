import React from 'react';
import { View } from 'react-native';
import Button from '../../../component/Button';
import { navigate } from '../../../navigation/NavigationService';
import styles from './styles';

class Login extends React.Component {
  state = {};

  render() {
    return (
      <View style={styles.container}>
        <Button onPress={() => navigate('Home')} title={'Login'} />
      </View>
    );
  }
}

export default Login;
