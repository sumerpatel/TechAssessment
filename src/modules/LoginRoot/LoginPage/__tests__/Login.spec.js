import React from 'react';
import { shallow } from 'enzyme';
import Login from '../Login';

describe('Login view render', () => {
  let wrapper;
  const props = {};
  beforeEach(() => {
    wrapper = shallow(<Login {...props} />);
  });

  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
