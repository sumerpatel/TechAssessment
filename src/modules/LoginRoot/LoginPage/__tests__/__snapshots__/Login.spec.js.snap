// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login view render should render correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Button
    onPress={[Function]}
    title="Login"
  />
</View>
`;
